src/guides/gelato-smart-contract-automation/index.md

```diff
@@ -0,0 +1,449 @@
+    // nothing to refund if no renter
+    uint256 refund = 0;
+    if (listing.user != address(0)) {
+        refund = ((listing.expires - block.timestamp) / 60 / 60 / 24 + 1) * listing.pricePerDay;
+        require(msg.value >= refund, "Not enough ether to cover refund");
+        payable(listing.user).transfer(refund);
+    }
+    // clean up data
+    IERC4907(nftContract).setUser(tokenId, address(0), 0);
+    EnumerableSet.remove(_nftContractTokensMap[nftContract], tokenId);
+    delete _listingMap[nftContract][tokenId];
+    if (EnumerableSet.length(_nftContractTokensMap[nftContract]) == 0) {
+        EnumerableSet.remove(_nftContracts, nftContract);
+    }
+    _nftsListed.decrement();
+
+    emit NFTUnlisted(
+        msg.sender,
+        nftContract,
+        tokenId,
+        refund
+    );
+```
+
+
+
+### Deploy the RentablePets contract to Goerli
+
+First, let's deploy the `RentablePets` contract to Goerli. If you haven't already done so, clone the [`unleashed_rentable_nft` repo](https://github.com/trufflesuite/unleashed_rentable_nft). Then, let's deploy it to the Goerli testnet using Truffle dashboard! To do so, let's add our Infura Goerli endpoint to our MetaMask wallet. First get your Infura endpoint from your Infura dashboard, and then navigate to Add Networks in MetaMask.
+
+![Infura endpoint](./img/infura_dashboard.png)
+
+![MetaMask wallet](./img/goerlidev.png)
+
+Make sure your MetaMask wallet is connected to `goerlidev` and run:
+
+```shell
+truffle dashboard
+truffle migrate --network dashboard
+```
+
+This should bring up Truffle dashboard on localhost:24012 and prompt you to sign the deployment. After that's been deployed, now let's verify it!
+
+### Deploy to Goerli
+
+### Deploy the RentablePets contract to Goerli
+
+Let's deploy the contract to Goerli. Then, let's deploy it to the Goerli testnet using Truffle dashboard! To do so, let's add our Infura Goerli endpoint to our MetaMask wallet. First get your Infura endpoint from your Infura dashboard, and then navigate to Add Networks in MetaMask.
+
+![Infura endpoint](./img/infura_dashboard.png)
+
+![MetaMask wallet](./img/goerlidev.png)
+
+Make sure your MetaMask wallet is connected to `goerlidev` and run:
+
+```shell
+truffle dashboard
+truffle migrate --network dashboard
+```
+
+This should bring up Truffle dashboard on localhost:24012 and prompt you to sign the deployment.
+
+## Add the task to Gelato Ops
+
+Gelato has a super easy to use UI to automate your tasks. Navigate to the [app](https://app.gelato.network/) and hit create task! It's pretty simple
+
+First, add your `Marketplace` contract address - because we didn't verify the contract, we'll have to manually add in the `abi`. To get this, go to `Marketplace.json` file in your `contracts_build_directory`. Alternatively, if you have the VS Code extension, you can directly access it there.
+
+![VS Code deployment](./img/deployments_vscode.png)
```

Comment:
Noticing that some of the embedded images are pretty large. Note that we have `attr_list` enabled so can add additional attributes to the markdown as follows:

```
![VS Code deployment](./img/deployments_vscode.png){: style="width:12rem"}
```